src/common/configs.ts

```diff
@@ -67,6 +67,8 @@ export interface ApiContext {
   chainId: SupportedChainId
   env: CowEnv
   baseUrls?: ApiBaseUrls
+  limiterOpts?: RateLimiterOpts
```

Comment:
This is added here but not used anywhere in the PR.
Is it need?

Reply:
<img width="1025" alt="image" src="https://github.com/cowprotocol/cow-sdk/assets/7122625/1f85f857-3a73-41fd-9b3a-69914e0f3927">

It is used here

Reply:
But this wasn't introduced in this PR.
Was it just not properly typed before?

Reply:
Sorry, good point actually!

Added a fix: https://github.com/cowprotocol/cow-sdk/pull/208/commits/826f4101b648c07b9d03f06d60772ed1bfa8ba1b